Source/WebKit/Platform/IPC/Connection.cpp

```diff
@@ -306,6 +306,15 @@ static HashMap<uintptr_t, HashMap<uint64_t, CompletionHandler<void(Decoder*)>>>&
     return map.get();
 }
 
+static void addAsyncReplyHandler(Connection& connection, uint64_t identifier, CompletionHandler<void(Decoder*)> completionHandler)
+{
+    Locker locker { asyncReplyHandlerMapLock };
+    auto result = asyncReplyHandlerMap().ensure(reinterpret_cast<uintptr_t>(&connection), [] {
```

Comment:
Thanks! I landed with the existing behavior, not changing it. Hopefully sometime in near future I'll remove the hash map (the data is just normal instance data), so that should take care of the ensure.